scripts/scr_planetary_feature/scr_planetary_feature.gml

```diff
@@ -1,7 +1,264 @@
-function scr_planetary_feature(argument0) {
+enum P_features {
+			Sororitas_Cathedral,
+			Necron_Tomb,
+			Artifact, 
+			STC_Fragment,
+			Ancient_Ruins,
+			Cave_Network,
+			Recruiting_World, 
+			Monastery,
+			Warlord6,
```

Comment:
better name?

Reply:
are you referring to warlord 6? because i'm just reusing the standard naming convention for faction diplomacy, so yeah will change but needs a more systmic work through of factions